src/test/resources/updates/jenkins.plugins.nodejs.tools.NodeJSInstaller.json

```diff
@@ -1,2277 +1,2284 @@
-{"list": [
```

Comment:
Too many changes due reformatting, i'm not able to review changes

Reply:
reverted the json file to the previous version and added the entries.
Might be good to reformat the file to correct the indentation

Reply:
After couple of commits, the review should now be clear. Seems the idea does automatic indentation upon paste.

Reply:
You need just to download the updated file from http://updates.jenkins-ci.org/updates/hudson.plugins.nodejs.tools.NodeJSInstaller.json and remove the non JSON part `downloadService.post('hudson.plugins.nodejs.tools.NodeJSInstaller',` from begin and `)` from end without apply any formatting